packages/zoltan2/src/algorithms/order/Zoltan2_AlgRCM.hpp

```diff
@@ -140,9 +141,9 @@ class AlgRCM : public Algorithm<Adapter>
     size_t count = 0; // CM label, reversed later
     size_t next = 0;  // next unmarked vertex
     Teuchos::Array<std::pair<gno_t, offset_t> >  children; // children and their degrees
-  
+
     while (count < nVtx) {
-  
+
       // Find suitable root vertex for this component.
       // First find an unmarked vertex, use to find root in next component.
       while ((next < nVtx) && (static_cast<Tpetra::global_size_t>(invPerm[next]) != INVALID)) next++;
```

Comment:
I'm surprised this doesn't warn about extra parentheses, but that's how the code was....

Reply:
Agreed, looks like a parentheses bug to me.